spec/controllers/media_controller_spec.rb

```diff
@@ -79,4 +85,43 @@
       end
     end
   end
+
+  describe '#verify_token' do
+    let(:id) { 'ab123cd4567' }
+    let(:file_name) { 'interesting_video.mp4' }
+    let(:user_ip_addr) { '192.168.1.100' }
+    let(:token) { StacksMediaToken.new(id, file_name, user_ip_addr) }
+
+    # rubocop:disable Metrics/LineLength
```

Comment:
personally, I hate inline rubocop exceptions.  Can the test be written in a way to avoid the long lines?   if we MUST have them, can the inline exceptions be limited to only those tests that really need them?   Or can we use a file exception in rucobop.yml?    @jkeck probably has a differing opinion.